src/neural/CL/OpenCL.cc

```diff
@@ -1,527 +1,183 @@
 
 const std::string sourceCode_sgemm =
 #include "clblast_level3/common.opencl"
 #include "clblast_level3/xgemm_part1.opencl"
 #include "clblast_level3/xgemm_part2.opencl"
 #include "clblast_level3/xgemm_part3.opencl"
 #include "clblast_level3/xgemm_batched.opencl"
-;
+    ;
 
 const std::string sourceCode_sgemv =
 #include "clblast_level3/xgemv.opencl"
-;
+    ;
 
 thread_local ThreadData opencl_thread_data;
 
 void OpenCL::ensure_thread_initialized() {
   if (!opencl_thread_data.m_is_initialized) {
     // Make kernels
-    opencl_thread_data.m_convolve1_kernel =
-    cl::Kernel(m_program, "convolve1");
-    opencl_thread_data.m_merge_kernel =
-    cl::Kernel(m_program, "merge_bn");
+    opencl_thread_data.m_convolve1_kernel = cl::Kernel(m_program, "convolve1");
+    opencl_thread_data.m_merge_kernel = cl::Kernel(m_program, "merge_bn");
     opencl_thread_data.m_in_transform_kernel =
-    cl::Kernel(m_program, "in_transform");
-    opencl_thread_data.m_sgemm_kernel =
-    cl::Kernel(m_program, "XgemmBatched");
+        cl::Kernel(m_program, "in_transform");
+    opencl_thread_data.m_sgemm_kernel = cl::Kernel(m_program, "XgemmBatched");
     opencl_thread_data.m_out_transform_bn_kernel =
-    cl::Kernel(m_program, "out_transform_fused_bn");
+        cl::Kernel(m_program, "out_transform_fused_bn");
     opencl_thread_data.m_out_transform_bn_in_kernel =
-    cl::Kernel(m_program, "out_transform_fused_bn_in");
-    opencl_thread_data.m_sgemv_kernel =
-    cl::Kernel(m_program, "Xgemv");
-    opencl_thread_data.m_commandqueue =
-    cl::CommandQueue(m_context, m_device);
+        cl::Kernel(m_program, "out_transform_fused_bn_in");
+    opencl_thread_data.m_sgemv_kernel = cl::Kernel(m_program, "Xgemv");
+    opencl_thread_data.m_commandqueue = cl::CommandQueue(m_context, m_device);
     opencl_thread_data.m_is_initialized = true;
   }
 }
 
-void OpenCL_Network::add_weights(size_t layer,
-                                 size_t size,
-                                 const float * weights) {
+void OpenCL_Network::add_weights(size_t layer, size_t size,
+                                 const float* weights) {
   if (layer >= m_layers.size()) {
     m_layers.push_back(Layer());
   }
-  
+
   auto converted_weights = std::vector<net_t>();
-  for(auto i = size_t{0}; i < size; i++) {
+  for (auto i = size_t{0}; i < size; i++) {
     converted_weights.emplace_back(weights[i]);
   }
-  
+
   auto weightSize = size * sizeof(decltype(converted_weights)::value_type);
   m_layers.back().weights.emplace_back(
-                                       m_opencl.m_context,
-                                       CL_MEM_COPY_HOST_PTR | CL_MEM_READ_ONLY,
-                                       weightSize,
-                                       const_cast<net_t*>(converted_weights.data()));
+      m_opencl.m_context, CL_MEM_COPY_HOST_PTR | CL_MEM_READ_ONLY, weightSize,
+      const_cast<net_t*>(converted_weights.data()));
```

Comment:
In reference to my previous comment on different PR, if it *has* to be on more than one line, than only one argument per line is preferable IMO.

Reply:
It's clang-format google.

Reply:
The point of this PR is to pass clang-format on the OpenCL.cc file.